src/Bootstrap/LoadEnvironmentVariables.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+namespace Longman\TelegramBot\Bootstrap;
+
+use Dotenv\Dotenv;
+use Dotenv\Environment\Adapter\EnvConstAdapter;
+use Dotenv\Environment\Adapter\PutenvAdapter;
+use Dotenv\Environment\Adapter\ServerConstAdapter;
+use Dotenv\Environment\DotenvFactory;
+use Exception;
+use Longman\TelegramBot\Application;
+
+class LoadEnvironmentVariables
+{
+    public function bootstrap(Application $app)
+    {
+        try {
+            $this->createDotenv($app)->load();
+        } catch (Exception $e) {
+            $this->writeErrorAndDie($e);
```

Comment:
Does this make sense in respect of having this library integrated as part of an underlying project?
This literally forces the user to have a `.env` file (with that name!) in the current folder.

If I have a custom `whatever.env` that I load in my project, then the PHP Telegram Bot should just pick up those variables.

This bootstrapper should only be run when using the console command I think.

Reply:
> If I have a custom whatever.env that I load in my project, then the PHP Telegram Bot should just pick up those variables.

Reply:
Well, that's what the checks in the Telegram constructor are for, right?
If no API key is given, then an exception is thrown.

Same thing happens for the DB.

I think libraries that extend functionality of a project shouldn't require an own .env setup, but instead should define which environment variables are necessary for it to work.

Reply:
@noplanman when user will create an instance of out `Telegram/Application`, our script will check the project's root for `.env` file.

> I think libraries that extend functionality of a project shouldn't require an own .env setup, but instead should define which environment variables are necessary for it to work.

How to achieve this? Do not parse .env and just try to find variables in the $_ENV? Then user should parse .env file himself